packages/wdio-local-runner/src/processProxy.ts

```diff
@@ -0,0 +1,19 @@
+import { AsyncLocalStorage } from 'node:async_hooks'
+
+const originalProcess = process
+const processLocals = new AsyncLocalStorage()
+export const patchedProcess = new Proxy(
```

Comment:
Can you explain your intention with this?

Reply:
The main purpose is to be able to "intercept" some methods and properties of the process object like `env`, `send`, `stdout` ... so that while using the same process each "thread" has it's own values. It's being used in packages/wdio-local-runner/src/runLite.ts:56. This way there are other places in the wdio code base (like in the runner where process.send) which are not impacted by this feature.